test/cmd/builds.sh

```diff
@@ -138,16 +139,27 @@ echo "start-build: ok"
 os::test::junit::declare_suite_end
 
 os::test::junit::declare_suite_start "cmd/builds/cancel-build"
-os::cmd::expect_success_and_text "oc cancel-build ${started} --dump-logs --restart" "Restarted build ${started}."
+os::cmd::expect_success_and_text "oc cancel-build ${started} --dump-logs --restart" "restarted build \"${started}\""
 os::cmd::expect_success 'oc delete all --all'
 os::cmd::expect_success 'oc process -f examples/sample-app/application-template-dockerbuild.json -l build=docker | oc create -f -'
 os::cmd::try_until_success 'oc get build/ruby-sample-build-1'
 # Uses type/name resource syntax to cancel the build and check for proper message
-os::cmd::expect_success_and_text 'oc cancel-build build/ruby-sample-build-1' 'Build ruby-sample-build-1 was cancelled.'
+os::cmd::expect_success_and_text 'oc cancel-build build/ruby-sample-build-1' 'build "ruby-sample-build-1" cancelled'
 # Make sure canceling already cancelled build returns proper message
-os::cmd::try_until_text 'oc cancel-build build/ruby-sample-build-1' 'A cancellation event was already triggered for the build ruby-sample-build-1.'
+os::cmd::expect_success 'oc cancel-build build/ruby-sample-build-1'
+# Cancel all builds from a build configuration
+os::cmd::expect_success "oc start-build bc/ruby-sample-build"
+os::cmd::expect_success "oc start-build bc/ruby-sample-build"
+lastbuild=$(oc start-build bc/ruby-sample-build)
+os::cmd::expect_success_and_text 'oc cancel-build bc/ruby-sample-build', "build \"${lastbuild}\" cancelled"
+os::cmd::expect_success_and_text "oc get builds ${lastbuild} -o template --template '{{.status.phase}}'", 'Cancelled'
+os::cmd::expect_success 'oc cancel-build bc/ruby-sample-build'
```

Comment:
do this after you loop through and check the status of the builds.  otherwise you're running cancel twice before you check the status, which seems too generous for a test.